tests/testthat/test-coveralls.R

```diff
@@ -153,4 +153,4 @@ test_that("coveralls can spawn a job using repo_token - travis-pro #285", {
       expect_equal(json$git$head$message, "f")
     )
   )
-})
\ No newline at end of file
+})
```

Comment:
this newline was added automatically by my editor. If you'd like me to revert it, let me know.